README.md

```diff
@@ -442,23 +444,78 @@ We have a small collection of algorithms, one for every day of the month. Scroll
     - As you see, each input is mapped to an output of fixed size.
     - Now, implement a hash function that takes a string of variable length and returns an integer with a fixed maximum number of digits. Try to come up with your own method of returning a fixed length hash. The simplest method would be to just use a 3-digit length of the string. 001, 042, 340 for 1, 42 and 340 characters respectively. But that's no fun.
     - **Hint:** Try something with the ASCII values of each word maybe.
-    
+
   - **_Optional Problem_**
     - Hash functions may return the same output for various inputs. This is known as collision. There are two main collision resolution techniques:
       - [Separate Chaining](https://youtu.be/_xA8UvfOGgU)
       - [Open Addressing](https://youtu.be/Dk57JonwKNk)
     - Implement a hash table with one of the collision resolution techniques mentioned above.
-    
+
   - **Uses**
     - As mentioned above, hash functions are often used in combination with a hash table for rapid data lookup.
     - They are also useful in cryptography. A cryptographic hash function allows one to easily verify that some input data maps to a given hash value, but if the input data is unknown, it is deliberately difficult to reconstruct it (or any equivalent alternatives) by knowing the stored hash value.
     - Cryptographic hash functions also play a crucial role in blockchains.
-    
+
   - **Resources**
     - [Hash function – Wikipedia](https://en.wikipedia.org/wiki/Hash_function)
     - [Hash table - Wikipedia](https://en.wikipedia.org/wiki/Hash_table)
     - [An in-depth video about cryptocurrencies, blockchain and hash functions](https://youtu.be/bBC-nXj3Ng4)
-    
+
+
+### **December 20 - Dijkstra's Shortest Reach**
```

Comment:
Shortest Path* no?

Reply:
Both **Dijkstra's Shortest Reach** and **Dijkstra's Shortest Path** mean the same.